cardano-node-emulator/src/Cardano/Node/Emulator/LogMessages.hs

```diff
@@ -51,7 +52,7 @@ instance Pretty TxBalanceMsg where
           [ hang 2 $ vsep ["Balancing an unbalanced transaction:", pretty tx]
           , hang 2 $ vsep $ "Utxo index:" : (pretty <$> Map.toList utxo)
           ]
-    FinishedBalancing tx -> hang 2 $ vsep ["Finished balancing:", pretty tx]
+    FinishedBalancing tx _ -> hang 2 $ vsep ["Finished balancing:", pretty tx]
```

Comment:
no need to `log` the CardanoAddress?

Reply:
No need, since you usually know which it has to be, but on second thought it makes it easier to see which outputs go back to the wallet, so I will add it, thanks.